### PR TITLE
Delete item from cart

### PR DESCRIPTION
**Pull request by Yoyo2Code**
* Items can be removed from cart

### CODE REVIEW

File: spec/good_tests/models/property_spec.rb [hunk 1]
```diff
@@ -7,6 +7,8 @@
     it { is_expected.to validate_presence_of(:price_per_guest) }
```
Comment (Automatic365) [inline spec/good_tests/models/property_spec.rb]: very thorough. validations should be pretty set.

File: app/controllers/businesses_controller.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class BusinessesController < ApplicationController
+  def new
+    @business = Business.new
+  end
+
+  def create
+    business = Business.new(business_params)
+    if business.save!
+      RoleDelegator.new(business).assign_role('pending')
```
Comment (dtinianow) [inline app/controllers/businesses_controller.rb]: Revisit roledelegator after adding business admin functionality

File: app/models/role_delegator.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+class RoleDelegator 
+  attr_reader :record
+  def initialize(record)
+    @record = record
+  end
+
+  def assign_role(status)
```
Comment (dtinianow) [inline app/models/role_delegator.rb]: This seems like a good solution for dealing with multiple roles on different tables

File: db/seeds.rb [hunk 4]
```diff
@@ -1,24 +1,62 @@
...
+      end
+    end
+  end
+
+  def seed_users
+    User.create!(username: 'Yoseph', password: 'password')
+    User.first.orders.create!
+    User.first.orders.first.reservations.create!(starting_date: DateTime.new(2016, 8, 22), end_date: DateTime.new(2016, 8, 30), number_of_guests: 2, price: 1500, property_id: Property.first.id)
+  end
+end
```
Comment (dtinianow) [inline db/seeds.rb]: Nice work on seeding.  This looks great

File: spec/good_tests/models/cart_spec.rb [hunk 5]
```diff
@@ -39,33 +46,45 @@
...
-
   xit "should be able to find items" do
     cat1 = Location.create!(title: "arms")
-    item = cat1.items.create!(title: "Robot Arm", description: "Cool ass arm", price: 10000.0, image_path: 'http://img09.deviantart.net/588b/i/2004/272/7/2/i__robot_arm_by_chainsawdeathriot.jpg')
+    item = cat1.items.create!(
+      title: "Robot Arm",
+      description: "Cool ass arm",
+      price: 10_000.0,
+      image_path: 'http://img09.deviantart.net/588b/i/2004'\
+      '/272/7/2/i__robot_arm_by_chainsawdeathriot.jpg')
```
Comment (dtinianow) [inline spec/good_tests/models/cart_spec.rb]: Need to revisit these model tests to update them and remove old ones

**Comment by dtinianow** [comment]
Includes ability to delete item from cart
Adds root path, with ability to toggle between list of businesses and cities

**Comment by Yoyo2Code** [comment]
Hound errors will be worked on.

**Pull request merged**